demonstrations/how_to_use_qiskit1_with_pennylane.py

```diff
@@ -263,7 +263,8 @@ def qiskit_GHZ_circuit(n):
 pl_qfunc = qml.from_qiskit(qc, measurements=measurements)
 
 pl_circuit = qml.QNode(pl_qfunc, device=qml.device("default.qubit", wires=n))
-print(pl_circuit(shots=5))
+pl_circuit = qml.set_shots(pl_circuit, shots=5)
+print(pl_circuit())
```

Comment:
```suggestion
print(qml.set_shots(pl_circuit, shots=5))
```
Would rather not overwrite the `pl_circuit` variable definition.

Reply:
I'll update it with `()`